1.7.0

```diff
@@ -14,6 +14,42 @@
           }
         }
       ]
+    },
+    {
+      "sObjectName": "FinServ__FinancialAccount__c",
+      "description": "All Record Types- Rework Status Field Values",
+      "taskList": [
+        {
+          "jiraTicket": "ac-3853",
+          "description": "Remove unwanted picklist options from status field",
+          "actionType": "PicklistValueCustom",
+          "actionTypeName": "FinServ__Status__c",
+          "optionsMap": {
+              "picklistValues": "Await 1035 Funds,Held,Held At AE, Paid,Pre-Screen,Received,Returned,Reversed,Withdrawn",
+              "operation": "deactivate"
```

Comment:
Some of these aren't de-activating because of casing. But this list of deactivated values doesn't match what is in the ticket under item 1.b